src/libsync/syncengine.cpp

```diff
@@ -586,7 +586,11 @@ void SyncEngine::startSync()
     if (!_discoveryPhase->_remoteFolder.endsWith('/'))
         _discoveryPhase->_remoteFolder+='/';
     _discoveryPhase->_syncOptions = _syncOptions;
-    _discoveryPhase->_shouldDiscoverLocaly = [this](const QString &s) { return shouldDiscoverLocally(s); };
+    _discoveryPhase->_shouldDiscoverLocaly = [this](const QString &s) {
```

Comment:
nitpick: Please avoid single letter variables if they are used beyond a single line :)

Reply:
Esepcailly since `s` is in no way indicative of the thing the variable represents, a path